openshift/Dockerfile

```diff
@@ -32,10 +32,10 @@ RUN yum install -y \
  && yum clean all \
  && rm -rf /var/cache/yum
 
-COPY --from=builder /go/src/github.com/openshift/ansible-operator-plugins/build/ansible-operator /usr/local/bin/ansible-operator
-COPY release/ansible/ansible_collections ${HOME}/.ansible/collections/ansible_collections
+COPY --from=builder /go/src/github.com/openshift/ansible-operator-plugins/ansible-operator /usr/local/bin/ansible-operator
+COPY openshift/release/ansible/ansible_collections ${HOME}/.ansible/collections/ansible_collections
```

Comment:
Are the changes specific to downstream? It'd help to keep lower number of carry PRs for the future rebases.